NGitLab.Tests/Docker/GitLabTestContext.cs

```diff
@@ -18,7 +19,13 @@ namespace NGitLab.Tests.Docker
 {
     public sealed class GitLabTestContext : IDisposable
     {
-        private static readonly Policy s_gitlabRetryPolicy = Policy.Handle<GitLabException>().WaitAndRetry(10, _ => TimeSpan.FromSeconds(1));
+        private const int _maxRetryCount = 10;
+        private static readonly Policy s_gitlabRetryPolicy = Policy.Handle<GitLabException>()
+            .WaitAndRetry(_maxRetryCount, _ => TimeSpan.FromSeconds(1), (exception, timespan, retryCount, context) =>
+            {
+                TestContext.Progress.WriteLine($"{exception.Message} -> Polly Retry {retryCount} of {_maxRetryCount}...");
```

Comment:
You should use TestContext.Out to associate the log to the test as we don't need to see the message immediatly in the console.

Reply:
Made use of `TestContext.WriteLine`. I also output the current test's full name.